.github/workflows/7.17.yml

```diff
@@ -39,7 +39,11 @@ jobs:
         rake bundle:install
     - name: elasticsearch
       run: cd elasticsearch && bundle exec rake test:all
-    - name: elasticsearch-transport
+    - name: elasticsearch-transport faraday1
+      run: cd elasticsearch-transport && bundle install && bundle exec rake test:all
```

Comment:
```suggestion
      run: cd elasticsearch-transport && bundle install && bundle exec rake test:faraday1:all
```

Reply:
I purposefully did not add the `faraday1` namespace as it's not needed given that we are already setting the `BUNDLE_GEMFILE` env var, but no problem on adding it 👍